docs/csharp/quick-starts/numbers-in-csharp-local.md

```diff
@@ -0,0 +1,373 @@
+You can get the **remainder** by using the **modulo** operator, the `%` character. Try the following code in your `Main` method:
+
+```csharp
+int a = 7;
+int b = 4;
+int c = 3;
+int d = (a  + b) / c;
+int e = (a + b) % c;
+Console.WriteLine($"quotient: {d}");
+Console.WriteLine($"remainder: {e}");
+```
+
+The C# integer type differs from mathematical integers in one other
+way: the `int` type has minimum and maximum limits. Add this code to
+your `Main` method to see those limits:
+    
+```csharp
+int max = int.MaxValue;
+int min = int.MinValue;
+Console.WriteLine($"The range of integers is {min} to {max}");
+```
+
+If a calculation produces a value that exceeds those limits, you
+have an **underflow** or **overflow** condition. The answer appears
+to wrap from one limit to the other. Add these two lines to your `Main`
+method to see an example:
+
+```csharp
+int what = max + 3;
+Console.WriteLine($"An example of overflow: {what}");
+```
+    
+Notice that the answer is very close to the minimum (negative) integer. It's
+the same as `min + 2`. 
+The addition operation **overflowed** the allowed values for integers.
+The answer is a very large negative number because an overflow "wraps around"
+from the largest possible integer value to the smallest.
+
+There are other numeric types with different limits and precision that you
+would use when the `int` type doesn't meet your needs. Let's explore those next.
+
+Once again, let's move the code you wrote in this section into a separate method. Name it `TestLimits`. 
+
+## Work with the double type
+
+The `double` numeric type represents a double-precision floating point
+number. Those terms may be new to you. A **floating point** number is
+useful to represent non-integral numbers that may be very large, or small
+in magnitude. **Double-precision** means that these numbers are stored
+using greater precision than **single-precision**. On modern computers,
+it is more common to use double precision than single precision numbers.
+Let's explore. Try the following code in the interactive window and see the result:
+
+```csharp
+double a = 5;
+double b = 4;
+double c = 2;
+double d = (a  + b) / c;
+Console.WriteLine(d);
+```
+
+Notice that the answer includes the decimal portion of the quotient. Try a slightly
+more complicated expression with doubles:
+
+```csharp
+double e = 19;
+double f = 23;
+double g = 8;
+double h = (e  + f) / g;
+Console.WriteLine(h);
+```
+
+The range of a double value is much greater than integer values. Try the following
+code below what you've written so far:
+
+```csharp
+double max = double.MaxValue;
+double min = double.MinValue;
+Console.WriteLine($"The range of double is {min} to {max}");
+```
+
+These values are printed out to represent scientific notation. The number to
+the left of the `E` is the significant. The number to the right is the exponent,
+as a power of 10. Internally, these numbers are represented similarly, but in 
```

Comment:
I think that you're saying that the binary representation is similar to its representation in scientific notation. There are differences, though, and the implication of introducing it here is that it's true for floating-point numbers represented in scientific notation, but not of floating-point numbers not in scientific notation. Personally, I think that this is outside the scope of a beginning topic, so I'd delete it.

Reply:
good point.